tests/python/ConfigTest.py

```diff
@@ -773,83 +775,287 @@ def test_virtual_display(self):
-        self.assertEqual("raw", 
+        self.assertEqual("raw",
                          cfg.getVirtualDisplayViewColorSpaceName(view_name))
         self.assertEqual("", cfg.getVirtualDisplayViewLooks(view_name))
         self.assertEqual("", cfg.getVirtualDisplayViewRule(view_name))
         self.assertEqual("", cfg.getVirtualDisplayViewDescription(view_name))
 
         view_name = cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)[1]
         self.assertEqual("Film", view_name)
-        self.assertEqual("display_vt", 
+        self.assertEqual("display_vt",
                          cfg.getVirtualDisplayViewTransformName(view_name))
-        self.assertEqual(OCIO.OCIO_VIEW_USE_DISPLAY_NAME, 
+        self.assertEqual(OCIO.OCIO_VIEW_USE_DISPLAY_NAME,
                          cfg.getVirtualDisplayViewColorSpaceName(view_name))
         self.assertEqual("", cfg.getVirtualDisplayViewLooks(view_name))
         self.assertEqual("", cfg.getVirtualDisplayViewRule(view_name))
         self.assertEqual("", cfg.getVirtualDisplayViewDescription(view_name))
 
         self.assertEqual(1, len(cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)))
-        self.assertEqual("sview2", 
+        self.assertEqual("sview2",
                          cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)[0])
 
         # Remove a view from the virtual display
         cfg.removeVirtualDisplayView("Raw")
 
         self.assertEqual(
-            1, 
+            1,
             len(cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)))
         self.assertEqual(
-            "Film", 
+            "Film",
             cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)[0])
 
         self.assertEqual(1, len(cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)))
-        self.assertEqual("sview2", 
+        self.assertEqual("sview2",
                          cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)[0])
 
         # Remove a shared view from the virtual display
         cfg.removeVirtualDisplayView("sview2")
         self.assertEqual(
-            1, 
+            1,
             len(cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)))
         self.assertEqual(0, len(cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)))
 
         cfg.addVirtualDisplaySharedView("sview2")
         self.assertEqual(
-            1, 
+            1,
             len(cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)))
         self.assertEqual(1, len(cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)))
 
         # Remove the virtual display
         cfg.clearVirtualDisplay()
         self.assertEqual(
-            0, 
+            0,
             len(cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)))
         self.assertEqual(0, len(cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)))
+
+
+class ConfigVirtualDisplayTest(unittest.TestCase):
+    def setUp(self):
+        self.cfg = OCIO.Config().CreateFromStream(SIMPLE_CONFIG_VIRTUAL_DISPLAY)
```

Comment:
```suggestion
        self.cfg = OCIO.Config.CreateFromStream(SIMPLE_CONFIG_VIRTUAL_DISPLAY)
```
`CreateFromStream` is a static method.

Reply:
Same comment below